weblate/utils/checks.py

```diff
@@ -270,27 +270,27 @@ def check_settings(app_configs, **kwargs):
     if not settings.ADMINS or "noreply@weblate.org" in (x[1] for x in settings.ADMINS):
         errors.append(
             weblate_check(
-                "weblate.E011", "The site admins seem to be wrongly configured", Error
+                "weblate.E011", "E-mail addresses for site admins is misconfigured", Error
```

Comment:
Or usernames?

Reply:
It's about this: https://docs.weblate.org/en/latest/admin/install.html#production-admins

So e-mail seems good fit here.